test/repository_test.rb

```diff
@@ -115,6 +126,18 @@
           UserRepository.create(user1)
           user1.id.must_equal id
         end
+
+        describe 'when entity is already persisted' do
+          it 'assigns and persists created_at attribute' do
+            result = PhoneRepository.create(@unpersisted_phone)
+            result.created_at.must_equal result.created_at
+          end
+
+          it 'does not touch created_at' do
```

Comment:
Please add another describe line here

describe 'when entity is not persisted' do